### ISSUE TITLE
Can I create my own glyphs ?

### ISSUE THREAD

**Issue by hewcaw** [ryanoasis/powerline-extra-symbols#15]
I want to create a box like with border (like Github Star button). Is it hard to make one ? Why there only few glyphs ? Thank you!

**Comment by extremepayne** [comment]
https://www.nerdfonts.com/#home
This repo holds only a few glyphs. Just a couple that the author thought would be fun when doing powerline-esque stuff. Nerd Fonts is the one you want, it comes with 3,600 icons. Including the GitHub star button, at \uf41e.

**Comment by hewcaw** [comment]
No, I want to make separator like \uE0B4, but like border-radius: 5-10px, can I make one and is it easy ? I don't know why he made something like the flame separators, I mean .. really ? Btw, happy new year bro =))

**Comment by extremepayne** [comment]
Ehh, flame separators are pretty cool. And happy new year to you, too. 
I'm not super into font creation, but my first guess is that a new glyph is not the right solution. It depends on the use case, but there might be an easier way. You can totally make a new glyph, but you'll have to re-patch the font yourself. My guess is patching a font is not easy.

**Comment by hewcaw** [comment]
Yeah, I thought like that too cause i didn't see much people do that. But I see at the end of https://www.nerdfonts.com/, there is a Font Patcher Script so maybe I think it'll be easy or at least not that hard, anyway thank you so much. 
I'm working on a solution for https://github.com/bagrat/vim-buffet, so I'll update this if there is any hope and also if you care about it, haha
